backend/gateway/stats/prometheus_reporter.go

```diff
@@ -0,0 +1,13 @@
+package stats
+
+import (
+	gatewayv1 "github.com/lyft/clutch/backend/api/config/gateway/v1"
+	tally_prom "github.com/uber-go/tally/prometheus"
```

Comment:
nit: I would stay away from using `_` in variable names.

Reply:
fixed